### PR TITLE
replace logic in hangfire

### PR DESCRIPTION
**Pull request by Atalay-Polat**
Deleted stored procedures and implemented new logic

### CODE REVIEW

File: exalearn-api/ExaLearn.Dal/Migrations/20210817111332_InitialDb.cs [hunk 1]
```diff
@@ -1217,17 +1217,16 @@ protected override void Up(MigrationBuilder migrationBuilder)
...
             migrationBuilder.Sql("CREATE PROCEDURE archiveexpiredassignedtest() language plpgsql AS $$ BEGIN UPDATE \"AssignTests\" SET \"IsExpired\" = true FROM \"PassedTests\" WHERE \"PassedTests\".\"Status\" = 4 AND \"PassedTests\".\"AssignTestId\" = \"AssignTests\".\"Id\"; END; $$");
-            migrationBuilder.Sql("CREATE PROCEDURE archivepassedassignedtest() language plpgsql AS $$ BEGIN UPDATE \"AssignTests\" SET \"Passed\" = true FROM \"PassedTests\" WHERE \"PassedTests\".\"Status\" = 2 AND \"PassedTests\".\"AssignTestId\" = \"AssignTests\".\"Id\"; END; $$");
+            migrationBuilder.Sql("CREATE PROCEDURE archivepassedassignedtest() language plpgsql AS $$ BEGIN UPDATE \"AssignTests\" SET \"Passed\" = true FROM \"PassedTests\" WHERE \"PassedTests\".\"Status\" = 2 AND \"PassedTests\".\"AssignTestId\" = \"AssignTests\".\"Id\"; END; $$");*/
         }
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Dal/Migrations/20210817111332_InitialDb.cs:1218]: why did you made comments?

File: exalearn-api/ExaLearn.Dal/Migrations/20210817111332_InitialDb.cs [hunk 2]
```diff
@@ -1217,17 +1217,16 @@ protected override void Up(MigrationBuilder migrationBuilder)
...
 
         protected override void Down(MigrationBuilder migrationBuilder)
         {
-            migrationBuilder.Sql("drop procedure archiveexpiredassigntest(TIMESPAN)");
+            /*migrationBuilder.Sql("drop procedure archiveexpiredassigntest(TIMESPAN)");
             migrationBuilder.Sql("drop procedure archiveexpiredassignedtest()");
-            migrationBuilder.Sql("drop procedure archivepassedassignedtest()");
-
+            migrationBuilder.Sql("drop procedure archivepassedassignedtest()");*/
             migrationBuilder.DropTable(
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Dal/Migrations/20210817111332_InitialDb.cs:1222]: also

File: exalearn-api/ExaLearn.WebApi/Startup.cs [hunk 3]
```diff
@@ -133,7 +133,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ExaLearn
             app.UseHangfireDashboard("/dashboard");
             app.UseHangfireServer();
 
-            RecurringJob.AddOrUpdate(() => assignTestRepository.ArchiveExpiredAssignTest(), Cron.Daily);
+            RecurringJob.AddOrUpdate(() => assignTestRepository.ArchiveExpiredAssignTest(), "*/15 * * * *");
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.WebApi/Startup.cs:137]: what is it? "*/15 * * * *"

Comment (Atalay-Polat) [reply exalearn-api/ExaLearn.WebApi/Startup.cs:137]: > what is it? "*/15 * * * *"
time interval in hangfire (15 min)

**Review by vlad12345123 — APPROVED** [review]

**Review by GeranSkot — APPROVED** [review]

**Pull request merged**